### ISSUE TITLE
Check that a Movie exists before downloading

### ISSUE THREAD

**Issue by UnR34L** [Radarr/Radarr#1840]
Radarr isn't checking if the Movies already exists before it downloads it, resulting is duplicate downloads and wasted bandwidth.

**Comment by galli-leo** [comment]
How are you adding movies to Radarr? Does radarr know about the existing movie files?

**Comment by UnR34L** [comment]
I have added movies that I want to Radarr but sometimes I'll manually download movies which go in the same directory and use the same folder/file name that Radarr expects.

I use Transmission so Radarr could check Transmission if it wanted to, I know that Sonarr does this. I believe Sonarr also periodically scans folders for changes this would solve it too.

I'd like Radarr to check the recent folders used from the bulk import feature or the path that was set when the movie was added, before it fetches a release.

Using the drone factory folder, telling the download client to drop movies in there should help for now, but I'd still prefer checks to be put in place.
